### ISSUE TITLE
[Featrue Request] CMake

### ISSUE THREAD

**Issue by OgreTransporter** [Melown/vts-backend#33]
It would be nice if the backend could also be built with CMake. So that you can also build it under Windows and macOS.

**Comment by vaclavblazek** [comment]
You mean `vts-backend`? It's just configuration, there's nothing to be built. Also, the configuration is Linux/nginx specific.

**Comment by OgreTransporter** [comment]
As I understand it, VTS-Backend is the superrepository of VTS-Mapproxy, VTS-VTSD and VTS-Tools which builds these programs. If this is not the case, I simply copy the ticket to all subprojects.

**Comment by vaclavblazek** [comment]
No, `vts-backend` is debian/ubuntu package that _depends_ on all other packages and provices nginx, vtsd and mapproxy configuration as an example and/or bootstrap for deployment.

NB: vtsd and mapproxy cannot be built on Windows! They are Linux specific. They _might_ be ported to Mac. They were never intended to be run on Windows. You might run them in WSL (Windows for Linux Subsystem).
